Update google clud logo in confidential computing WD-21719

muhammadbassiony opened the pull request:
## Done

- Update google colud logo in confidential computing

## Ticket

https://warthogs.atlassian.net/browse/WD-21719?linkSource=email

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-15049.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/13011)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

mtruj013 commented on templates/confidential-computing/index.html:
```diff
@@ -217,10 +217,10 @@ <h2>Launch enterprise-grade confidential VMs on all major public clouds</h2>
                   alt="Google Cloud logo",
-                  height="22",
-                  width="120",
+                  height="80",
+                  width="600",
```
Tried this sizing out locally and it seems to work:
```suggestion
                  height="27",
                  width="239",
```

mtruj013 commented on templates/confidential-computing/index.html:
```diff
@@ -217,10 +217,10 @@ <h2>Launch enterprise-grade confidential VMs on all major public clouds</h2>
           <div class="row">
             <div class="col-2 col-medium-3 col-small-2 u-pad-logo">
```
These should all be col-3, this is what's causing it to look extra small on desktop when you try to change the dimensions 
```suggestion
            <div class="col-3 col-medium-3 col-small-2 u-pad-logo">
```

mtruj013 reviewed (dismissed):
Thanks @muhammadbassiony, approved with some changes which should fix the scaling issue. For future BAU tickets please also link the jira ticket and target demo path in your pr description along with the appropriate review labels as described [here](https://webteam.canonical.com/practices/copy-doc-updates) (for updates where you were provided a copy doc and/or figma, please link that in the description as well). Thanks!

mattea-turic commented:
Thanks @muhammadbassiony !

As discussed on MM, I'm happy to add a design +1 following the resolution of the logo problem – thanks @mtruj013 for jumping in :)

muhammadbassiony commented:
> Thanks @muhammadbassiony, approved with some changes which should fix the scaling issue. For future BAU tickets please also link the jira ticket and target demo path in your pr description along with the appropriate review labels as described [here](https://webteam.canonical.com/practices/copy-doc-updates) (for updates where you were provided a copy doc and/or figma, please link that in the description as well). Thanks!

@mtruj013 appreciate the help and taking the time to look into this ticket, I've added the suggested changes
@mattea-turic can you also take a look

mattea-turic commented:
Hmmm the logo is still looking small for me on smaller desktops:
<img width="1022" alt="Screenshot 2025-04-29 at 10 51 54" src="https://github.com/user-attachments/assets/5449d516-9f1f-4c8b-8d57-ed6975c9aeba" />

And too big on larger tablets:
<img width="1010" alt="Screenshot 2025-04-29 at 10 52 34" src="https://github.com/user-attachments/assets/7b51bfe7-fecb-42b3-acf4-155ef37aae2b" />

mtruj013 commented:
@mattea-turic it's because demos are down, none of the changes @muhammadbassiony made are being reflected. Not sure how to go about design review in this situation really, but locally this still shows up squished. @muhammadbassiony can you try the values I recommended?

current: 
![Screenshot from 2025-04-29 14-47-34](https://github.com/user-attachments/assets/90796353-6c09-4a61-9679-367752290333)
![Screenshot from 2025-04-29 14-47-12](https://github.com/user-attachments/assets/ee71063e-310b-4fc3-9b8b-928f4c226846)

target: 
![Screenshot from 2025-04-29 14-48-54](https://github.com/user-attachments/assets/f9e48c6c-aa2a-4910-9e87-6db1155e4b39)
![Screenshot from 2025-04-29 14-48-41](https://github.com/user-attachments/assets/46166cad-a21d-467b-95e5-b4dd25dbd38e)

muhammadbassiony commented:
> @mattea-turic it's because demos are down, none of the changes @muhammadbassiony made are being reflected. Not sure how to go about design review in this situation really, but locally this still shows up squished. @muhammadbassiony can you try the values I recommended?
> 
> current: ![Screenshot from 2025-04-29 14-47-34](https://private-user-images.githubusercontent.com/17607612/438758624-90796353-6c09-4a61-9679-367752290333.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU5MzEzMjQsIm5iZiI6MTc0NTkzMTAyNCwicGF0aCI6Ii8xNzYwNzYxMi80Mzg3NTg2MjQtOTA3OTYzNTMtNmMwOS00YTYxLTk2NzktMzY3NzUyMjkwMzMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI5VDEyNTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJlNzg1ZTRkNjVhNGIwZDg1MmE2NjQ0MzlhZDRiMjM0OWEyNmU3MjVmYjI5YmYwZjE4ODAzNjU5NGFhMGJkZWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.9zIKJpXYTqjUKnvAxZTBX5rGLC_1XGmNMWRpBwz8UN8) ![Screenshot from 2025-04-29 14-47-12](https://private-user-images.githubusercontent.com/17607612/438758630-ee71063e-310b-4fc3-9b8b-928f4c226846.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU5MzEzMjQsIm5iZiI6MTc0NTkzMTAyNCwicGF0aCI6Ii8xNzYwNzYxMi80Mzg3NTg2MzAtZWU3MTA2M2UtMzEwYi00ZmMzLTliOGItOTI4ZjRjMjI2ODQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI5VDEyNTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwMWZhYjdmNTJiZjk5MTg0MzMzZGM0YzIxM2I4MDk4MWYzNTAzMmM5OWIxMTQxNDJiZjg1ZjBiYjQ1ODkwODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.5s8yYriJUorjeJ5fd5JDrxDuP-RvBYcLmxZCPebqZkg)
> 
> target: ![Screenshot from 2025-04-29 14-48-54](https://private-user-images.githubusercontent.com/17607612/438758971-f9e48c6c-aa2a-4910-9e87-6db1155e4b39.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU5MzEzMjQsIm5iZiI6MTc0NTkzMTAyNCwicGF0aCI6Ii8xNzYwNzYxMi80Mzg3NTg5NzEtZjllNDhjNmMtYWEyYS00OTEwLTllODctNmRiMTE1NWU0YjM5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI5VDEyNTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5MjE3ZmUyOGQwOTMzNjgzYWMxY2MxYjcwNTkxZWEzYThkODM4NmQ3MDFkOWY4YjNjZTYxYTVjN2E0M2E5NTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2M7dTFYOSW9dn4EWGfjb8RNHR_tc7wUM5dk08bFEnXY) ![Screenshot from 2025-04-29 14-48-41](https://private-user-images.githubusercontent.com/17607612/438758978-46166cad-a21d-467b-95e5-b4dd25dbd38e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDU5MzEzMjQsIm5iZiI6MTc0NTkzMTAyNCwicGF0aCI6Ii8xNzYwNzYxMi80Mzg3NTg5NzgtNDYxNjZjYWQtYTIxZC00NjdiLTk1ZTUtYjRkZDI1ZGJkMzhlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA0MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNDI5VDEyNTAyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU5MDg5ZDZiOWI4NjliMGZlN2M1ZWM2ZmRhYmQzMGY0NWI4ZTE3N2Q1ZDc4NmY2NmI1OWNlZmIzNzNmMGIxMTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Ak67589Tm6RRNIloQMxMSC7S8XJQlmXto8MCQypkd9w)

I changed to them, then thought Mattea's comment was regarding them so I changed them again 😅 reverting now

muhammadbassiony commented:
accidentally closed the PR so had to push a new commit, will squash merge later

mtruj013 commented:
demos are back @mattea-turic could you have a look [here](https://ubuntu-com-15049.demos.haus/confidential-computing)?

@muhammadbassiony thanks for all the changes, from a dev perspective LGTM!

mattea-turic commented:
@muhammadbassiony Looking good! Thanks!
